### DISCUSSION TITLE
Logging out & psuedo-anonymous users

### DISCUSSION THREAD

**Discussion by JordanMajd** [beakerbrowser/beaker#1565] [General]
Is there a way to not be signed in to a profile? Is there a way to log out of the current profile?

**Comment by pfrazee** [comment]
Not currently. FWIW the profile is just a hyperdrive that represents you -- it's not like you're signed into any service. (Feedback has been pretty clear: we obviously need to explain that better during setup.)

Eventually multi-profile support should be added.

**Comment by omgitsraven** [comment]
Does this mean there's no way to identify the creator of a page that you're viewing? (Or even to identify that two pages were created by the same user?)

**Reply by pfrazee** [reply]
ATM there's no metadata in the drives that indicate the author. I think something like that would be smart, but it should be something the author is conscious of.
